src/resolvers/get-model.ts

```diff
@@ -609,8 +683,26 @@ function getModelFromMakeDescription(make: Make, description: string): string |
       }
       break
     }
+    case Make.VOLVO: {
```

Comment:
Why do you have the spaces in front here? Are there other vehicles with a string that follow this pattern without a space?

Reply:
Its just to avoid potential double matchinng in the future for example if we would add a model that called `xv60`